sdk/communication/Azure.Communication.ProgrammableConnectivity/samples/DeviceNetworkSample.md

```diff
@@ -0,0 +1,15 @@
+# Network retrieve
+
+To get the Network name from a network identifier (IPv4/IPv6/NetworkCode), run the following code.
+
+```C# Snippet:APC_Sample_NetworkRetrievalTest
+string ApcGatewayId = "/subscriptions/abcdefgh/resourceGroups/dev-testing-eastus/providers/Microsoft.programmableconnectivity/gateways/apcg-eastus";
```

Comment:
The names should follow C# norms for variables.    

- `ApcGatewayId` => `apcGatewayId`   _(unless you make this a constant)_
- `_endpoint` => `endpoint`   _(not a class member)_
- `_credential` => `credential` _(not a class member)_